cea/optimization/master/cost_model.py

```diff
@@ -14,7 +14,7 @@
 import cea.technologies.solar.solar_collector as stc
 import numpy as np
 import pandas as pd
-
+from cea.optimization.optimization_constants import *
```

Comment:
we could actually just call the module `constants`, since it's full name is `cea.optimization.constants`